### PR TITLE
Rename unknown variables in vehicle motion code

### PR DESCRIPTION
**Pull request by spacek531**
Another housekeeping thing. Fingers crossed that replays pass.

The todos will create monster diffs because of the indentation changes.

### CODE REVIEW

**Comment by spacek531** [comment]
For some reason this seems to get hung up on the tests. I'm not sure why.

File: src/openrct2/ride/Vehicle.h [hunk 1]
```diff
@@ -583,6 +583,14 @@ enum
     SOUND_RANGE_NONE = 255
 };
 
+enum
+{
+    VEHICLE_MOVEMENT_TRANSLATE_X = 1 << 0,
```
Comment (duncanspumpkin) [inline src/openrct2/ride/Vehicle.h:588]: Follow the coding style for flags:
```cpp
namespace VehicleMovementTranslate
{
    constexpr uint8_t X = (1 << 0);
    ...
}
```

**Review by duncanspumpkin — COMMENTED** [review]
There is a lot of common code between these. Do you think there is much scope to extract this out into a function.

**Comment by spacek531** [comment]
I can probably move some of the common code into functions

File: src/openrct2/ride/CableLift.cpp [hunk 2]
```diff
@@ -230,7 +230,7 @@ bool Vehicle::CableLiftUpdateTrackMotionForwards()
     auto curRide = GetRide();
     if (curRide == nullptr)
         return false;
-
+    // why 13962? This is the SubpositionMovementDistances XYZ translation value plus 1
```
Comment (duncanspumpkin) [inline src/openrct2/ride/CableLift.cpp:233]: Suspect its the smallest distance that actually can move the vehicle in pixels.

**Comment by spacek531** [comment]
I marked this PR as ready so that the diff doesn't get too big. There's more I want to do, but in another PR because it will generate huge diffs.

File: src/openrct2/ride/Vehicle.cpp [hunk 3]
```diff
@@ -8206,6 +8241,7 @@ bool Vehicle::UpdateTrackMotionForwardsGetNewTrack(uint16_t trackType, Ride* cur
 bool Vehicle::UpdateTrackMotionForwards(rct_ride_entry_vehicle* vehicleEntry, Ride* curRide, rct_ride_entry* rideEntry)
 {
     uint16_t otherVehicleIndex = SPRITE_INDEX_NULL;
+    // TODO: replace this with a while loop or a for (;;_vehicleUnkF64E10++) loop
```
Comment (spacek531) [inline src/openrct2/ride/Vehicle.cpp:8244]: Will change indentation which will create a huge diff

File: src/openrct2/ride/Vehicle.cpp [hunk 4]
```diff
@@ -8318,45 +8352,32 @@ bool Vehicle::UpdateTrackMotionForwards(rct_ride_entry_vehicle* vehicleEntry, Ri
     // loc_6DB706
     const auto moveInfo = GetMoveInfo();
     trackType = GetTrackType();
+    // TODO: remove this redundant variable and remove the reduntant brackets
```
Comment (spacek531) [inline src/openrct2/ride/Vehicle.cpp:8355]: Will change indentation which will create a huge diff

File: src/openrct2/ride/Vehicle.cpp [hunk 5]
```diff
@@ -8647,29 +8669,14 @@ bool Vehicle::UpdateTrackMotionBackwards(rct_ride_entry_vehicle* vehicleEntry, R
 
         // loc_6DBD42
         track_progress = newTrackProgress;
+        // TODO: remove this redundant variable and remove the reduntant brackets
```
Comment (spacek531) [inline src/openrct2/ride/Vehicle.cpp:8672]: Will change indentation which will create a huge diff

**Review by duncanspumpkin — APPROVED** [review]

**Pull request merged**